asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/HttpRequestFactory.java

```diff
@@ -335,9 +364,30 @@ private static String toJsonString(Object objectToConvert) {
         }
     }
 
+
+    private static HttpEntity multipartEntity2(String attachmentFileName, InputStream attachmentContent, boolean notifyWatchers) {
```

Comment:
Can we get a better name for this method `multipartEntity2` ? Currently it only seems to communicate that this is a “second alternative” – but it's not clear/explicit how it's different from the first one…